core/store/src/lib.rs

```diff
@@ -37,6 +37,7 @@ pub use crate::trie::{
 };
 pub use flat_state::FlatStateDelta;
 
+mod cold_storage;
```

Comment:
Perhaps we wanna add `cold_storage` Cargo feature so this is not compiled in production code? The feature would be need to be implied by test_features so that we can run integration-tests.